options.c

```diff
@@ -3717,14 +3717,35 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 	{
 		.name	= "sync",
 		.lname	= "Synchronous I/O",
-		.type	= FIO_OPT_BOOL,
+		.type	= FIO_OPT_STR,
 		.off1	= offsetof(struct thread_options, sync_io),
 		.help	= "Use O_SYNC for buffered writes",
```

Comment:
This should probably change to "Use O_SYNC/O_DSYNC for buffered writes"

Reply:
Why is O_SYNC documented to be about buffered writes? As far as I understand both O_SYNC and O_DSYNC have effects for O_DIRECT style IO as well?